docs/tutorials/tutorial-unnest-datasource.md

```diff
@@ -0,0 +1,557 @@
+      "granularitySpec": {
+        "type": "uniform",
+        "queryGranularity": "NONE",
+        "rollup": false,
+        "segmentGranularity": "YEAR"
+      },
+      "timestampSpec": {
+        "column": "t",
+        "format": "auto"
+      },
+      "dimensionsSpec": {
+        "dimensions": [
+          "dim1",
+          "dim2",
+          "dim3"
+        ]
+      },
+      "metricsSpec": [
+        {
+          "name": "m1",
+          "type": "floatSum",
+          "fieldName": "m1"
+        },
+        {
+          "name": "m2",
+          "type": "doubleSum",
+          "fieldName": "m2"
+        }
+      ]
+    }
+  }
+}
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## View the data
+
+Now that the data is loaded, run the following query:
+
+```sql
+select * FROM nested_data
+```
+
+In the results, notice that the column named `dim3` has nested values like `["a","b"]`.  The example queries that follow unnest `dim3`  and run queries, such as scan. 
+
+## Unnest a single column
+
+The following section shows examples of how you can use the unnest datasource in queries. They all use the `nested_data` table you created earlier in the tutorial.
+
+### Scan query 
+
+The following native Scan query  also unnests the values in the `dim3` column by using the `unnest` datasource type:
+
+<details><summary>Show the query</summary>
+
+```json
+{
+  "queryType": "scan",
+  "dataSource": {
+    "type": "unnest",
+    "base": {
+      "type": "table",
+      "name": "nested_data"
+    },
+    "column": "dim3",
+    "outputName": "unnest-dim3",
+    "allowList": []
+  },
+  "intervals": {
+    "type": "intervals",
+    "intervals": [
+      "-146136543-09-08T08:23:32.096Z/146140482-04-24T15:36:27.903Z"
+    ]
+  },
+  "limit": 100,
+  "columns": [
+    "__time",
+    "dim1",
+    "dim2",
+    "dim3",
+    "m1",
+    "m2",
+    "unnest-dim3"
+  ],
+  "legacy": false,
+  "granularity": {
+    "type": "all"
+  },
+  "context": {
+    "debug": true,
+    "useCache": false
+  }
+}
+```
+s
+</details>
+
+In the results, you'll notice that there are now more rows than before and an additional column named `unnest-dim3`. The values of `unnest-dim3` are the same as the `dim3` column except the nested values have are no longer nested and are each a separate record. 
+
+With the `dataSource.allowList` parameter, you can unnest a subset of a column. Now, set the value of `allowList` to `["a","b"]` and run the query again. Only a subset of rows are returned based on the values you allowed.
+
+You can also implement filters. For example, you can add the following to the SCAN query to filter it:
+
+```json
+  "filter": {
+    "type": "in",
+    "dimension": "dim2",
+    "values": [
+      "a",
+      "abc",
+      ]
+  },
+```
+
+### Group by query
+
+The following query returns an unnested version of the column `dim3` as the column `unnest-dim3`, which is the `groupBy` dimension for the query too. The results are sorted in descending order. 
```

Comment:
Also could be helpful here to say what the groupBy query does in addition to returning an unnested version of `dim3`